blobfuse2-gen1-artifacts.yaml

```diff
@@ -0,0 +1,189 @@
+# Pipeline to build artifact for gen1 compliant adlsgen1fuse
+
+# Do not trigger this pipeline automatically
+trigger: none
+pr: none
+
+stages:
+
+  # ---------------------- Build adlsgen1fuse here and copy binary to artifact ------------------------------
```

Comment:
so we're always packaging adlsgen1fuse along with blobfuse? could you elaborate more on this pipeline

Reply:
We need to create a package that has both the binaries as blobfuse will just launch the other binary. This is not the standard package that we publish on linux repo or on github this is rather privately shared with the team who needs it from our pipeline artifacts.